src/App.js

```diff
@@ -1,6 +1,8 @@
 import React, { Component } from 'react';
 import logo from './logo.svg';
+import Text from './Text'
 import './App.css';
+import Previewer from './Previewer';
 
 class App extends Component {
```

Comment:
if you don't have state you can transform this components into a SFC - stateless function component